build/local_install_or_update

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Generic setup
+set -e
+
+# Ensure we have a $FLEPI_PATH
+if [ -z "${FLEPI_PATH}" ]; then
+    export FLEPI_PATH=$(realpath "$(pwd)")
+    if [ "$(basename "$FLEPI_PATH")" != "flepiMoP" ]; then 
+    echo "Could not find flepiMoP repository in your current directory."
+    exit 1
+    fi
+    echo "Using '$FLEPI_PATH' for \$FLEPI_PATH."
+fi
+
+# Test that flepiMoP is located there, exit if not
+if [ ! -d "$FLEPI_PATH" ]; then
+    echo "Could not find flepiMoP repository in your current directory. Please make sure you have correctly cloned flepiMoP in this directory."
+    exit 1
+fi
+
+# Create the conda environment if it doesn't exist
+if ! conda env list | grep -q "$FLEPI_CONDA"; then
+    conda env create --name flepimop-env --file $FLEPI_PATH/environment.yml
+fi
+
+# Ensure that conda environment is named flepimop-env
+if [ -z "${FLEPI_CONDA}" ]; then
+    export FLEPI_CONDA="flepimop-env"
+    echo "Using '$FLEPI_CONDA' for \$FLEPI_CONDA."
+fi
+
+# Activate the conda environment
+conda activate $FLEPI_CONDA
+
+# Check that the name of the FLEPI_CONDA variable matches the environment that has been set up, exit if not
+FLEPI_CONDA_ENV_MATCHES=$( conda info --envs | awk '{print $1}' | grep -x "$FLEPI_CONDA" | wc -l )
+if [ "$FLEPI_CONDA_ENV_MATCHES" -eq 0 ]; then
+    echo "Issue with conda environment: ensure you have set up your conda environment and it has the correct name."
+    exit 1
+fi
+
+# Load the conda environment
+eval "$(conda shell.bash hook)"
+conda activate $FLEPI_CONDA
```

Comment:
This seems to be in the wrong order/have the wrong logic. The
```bash
    conda env create --name flepimop-env --file $FLEPI_PATH/environment.yml
```
should be in side of the 
```bash
if [ "$FLEPI_CONDA_ENV_MATCHES" -eq 0 ]; then
    .....
fi
```
block? And then there shouldn't be two calls to `conda activate` right? I think we want the created env to have the name pulled from `$FLEPI_CONDA` right?

Reply:
Okay I re-downloaded `conda` then tested the script (with changes) locally. Everything seems to work. I think there were issues with my `conda` environment but it should be working correctly now. Ready again for a test on your end